packages/dashboard/package.json

```diff
@@ -5,15 +5,14 @@
     "@microsoft/load-themed-styles": "^1.7.13",
     "@types/react": "16.3.16",
     "@types/react-dom": "16.0.5",
-    "@types/react-grid-layout": "^0.16.4",
     "@uifabric/charting": "^0.4.0",
     "@uifabric/example-app-base": ">=6.5.1 <7.0.0",
     "@uifabric/experiments": ">=6.17.0 <7.0.0",
     "css-loader": "^0.15.1",
     "office-ui-fabric-react": ">=6.37.1 <7.0.0",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-grid-layout": "^0.16.6",
+    "react-grid-layout": "samiyaakhtar/react-grid-layout",
```

Comment:
> "react-grid-layout": "samiyaakhtar/react-grid-layout", [](start = 3, length = 55)

Don't need types anymore?

Reply:
Yes. I include typings file in the forked project since we made changes to it